src/components/select/_select.scss

```diff
@@ -121,14 +121,22 @@
   }
 
   .#{$prefix}--select--inline {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
+    display: grid;
+    grid-template-columns: auto auto;
+    display: -ms-grid;
```

Comment:
Did you see autoprefixer doesn't take care of generating `-ms-grid`?

Reply:
Line 126 actually shouldn't be there, I'll remove. IE11s version of grid doesn't support auto so we can't use it for this purpose. I have IE specific code on line 128 using flexbox that displays the select correctly, with the error message all the way left aligned.